update_discussions.sh

```
@@ -0,0 +1,34 @@
+set -e
+
+# Update 1: UI Graph Model (#58)
+gh api graphql -F id="D_kwDORxQzBc4AlQoO" -f body='### 🚀 Architectural Update: UI Baseline & Angular 21 (Packets 13-20)
+
+The front-end boundary has been significantly advanced recently:
+
+1. **Angular 21 Upgrade:** The `ui/` directory was sequentially upgraded to Angular 21, migrating to the new `@if`/`@for` block control flow and stabilizing tracking IDs (`track run.runId`).
+2. **Frontend Contract Seam:** A `FrontendDataService` abstraction was established with explicit mappers. The UI currently toggles between a `MockFrontendDataService` (for offline design) and an `ApiFrontendDataService` (backed by `HttpClient`) via environment files.
+3. **Desktop Skeleton:** A minimal Electron shell has been introduced in `desktop/` to host the web build.
+4. **Code Contract Guardrails:** A new validation script, `scripts/check_ui_code_contracts.py`, now runs during the `run_tests.sh ui` tier, explicitly enforcing top-of-file intent block headers across all TypeScript files.
+' -f query='mutation($id: ID!, $body: String!) { addDiscussionComment(input: {discussionId: $id, body: $body}) { comment { url } } }' > /dev/null
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)